pkg/serverless/aws/arn.go

```diff
@@ -46,13 +81,15 @@ func SetARN(arn string) {
 
 	arn = strings.ToLower(arn)
 
+	alias := ""
 	// remove the version if any
-	// format: arn:aws:lambda:<region>:<account-id>:function:<function-name>[:<version>]
 	if parts := strings.Split(arn, ":"); len(parts) > 7 {
 		arn = strings.Join(parts[:7], ":")
+		alias = strings.TrimPrefix(parts[7], "$")
```

Comment:
I think this is actually a "qualifier". A qualifier could be an alias, but it could also just be a version number. If the qualifier is just "7", it's not an alias.

https://docs.aws.amazon.com/lambda/latest/dg/configuration-aliases.html

Reply:
So, if that's the case, I would go through and replace references to "alias" accordingly.